chain/jsonrpc/jsonrpc-tests/res/genesis_config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "protocol_version": 79,
+  "protocol_version": 80,
```

Comment:
This is needed to make some jsonrpc test to pass, it was happening for every protocol upgrade so far.